### PR TITLE
Spelling corrections

### LINKED ISSUE #804: Please fix bad merge from #801
Several edits were made to an autogenerated file which will be lost on update. Comments are inline on the https://github.com/docker/distribution/pull/801.

### PR DESCRIPTION
**Pull request by RichardScothern**
Signed-off-by: Richard Scothern <richard.scothern@gmail.com>

closes #804

### CODE REVIEW

**Comment by stevvooe** [comment]
This is still causing a diff after `make docs/spec/api.md`:

```diff
diff --git a/docs/spec/api.md b/docs/spec/api.md
index c573178..12c1043 100644
--- a/docs/spec/api.md
+++ b/docs/spec/api.md
@@ -175,7 +175,6 @@ identify a set of modifications.
       <li>Added error code for unsupported operations.</li>
     </ul>
   </dd>
-
 </dl>
 
 ## Overview
@@ -727,7 +726,8 @@ delete may be issued with the following request format:
 
     DELETE /v2/<name>/blobs/<digest>
 
-If the blob exists and has been successfully deleted, the following response will be issued:
+If the blob exists and has been successfully deleted, the following response
+will be issued:
 
     202 Accepted
     Content-Length: None
@@ -735,6 +735,8 @@ If the blob exists and has been successfully deleted, the following response wil
 If the blob had already been deleted or did not exist, a `404 Not Found`
 response will be issued instead.
 
+If a layer is deleted which is referenced by a manifest in the registry,
+then the complete images will not be resolvable.
 
 #### Pushing an Image Manifest
 
@@ -1022,7 +1024,6 @@ A list of methods and URIs are covered in the table below:
 | PUT | `/v2/<name>/blobs/uploads/<uuid>` | Blob Upload | Complete the upload specified by `uuid`, optionally appending the body as the final chunk. |
 | DELETE | `/v2/<name>/blobs/uploads/<uuid>` | Blob Upload | Cancel outstanding upload processes, releasing associated resources. If this is not called, the unfinished uploads will eventually timeout. |
 | GET | `/v2/_catalog` | Catalog | Retrieve a sorted, json list of repositories available in the registry. |
-| DELETE | `/v2/<name>/blobs/<digest>` | Blob delete | Delete the blob identified by `name` and `digest`|
 
 
 The detail for each endpoint is covered in the following sections.
@@ -1732,7 +1733,6 @@ The error codes that may be included in the response body are enumerated below:
 
 #### DELETE Manifest
 
-
 Delete the manifest identified by `name` and `reference`. Note that a manifest can _only_ be deleted by `digest`.
 
```

**Comment by RichardScothern** [comment]
I see. descriptors.go is missing a section.

**Comment by RichardScothern** [comment]
descriptors.go has been updated.  PTAL @stevvooe

File: registry/api/v2/descriptors.go [hunk 1]
```diff
@@ -919,6 +918,70 @@ var routeDescriptors = []RouteDescriptor{
...
+								},
+							},
+							{
+								Description: "Delete is not enabled on the registry",
+								StatusCode:  http.StatusMethodNotAllowed,
+								Body: BodyDescriptor{
+									ContentType: "application/json; charset=utf-8",
+									Format:      errorsBody,
+								},
+								ErrorCodes: []errcode.ErrorCode{
```
Comment (stevvooe) [inline registry/api/v2/descriptors.go]: Small nit: I don't think we return `UNSUPPORTED` on method not allowed.

**Comment by stevvooe** [comment]
@RichardScothern Do we know why this dropped out? I am fairly sure we merged this correctly.

**Comment by stevvooe** [comment]
LGTM

**Comment by RichardScothern** [comment]
It's not clear from `git log` what happened.  It's possible it was in the first soft-delete PR (with tombstones), but missed from the link-removal version.

File: registry/api/v2/descriptors.go (hunk 1 above)
Comment (RichardScothern) [inline registry/api/v2/descriptors.go]: Confirmed:
```
curl -X DELETE http://localhost:5000/v2/mongo/blobs/sha256:5edccc13a9bd6722232ab891591bf3d19876c60787dce4e091f29bdf60192e98 
{"errors":[{"code":"UNSUPPORTED","message":"The operation is unsupported."}]}
```

**Pull request merged**